cmd/kubeapps-apis/plugin_test/utils.go

```diff
@@ -153,3 +160,23 @@ func MakePackageAppVersion(appVersion, pkgVersion string) *corev1.PackageAppVers
 		PkgVersion: pkgVersion,
 	}
 }
+
+func MakePackageRepositoryDetail(name string, plugin *plugins.Plugin) *corev1.PackageRepositoryDetail {
+	return &corev1.PackageRepositoryDetail{
+		PackageRepoRef: &corev1.PackageRepositoryReference{
+			Context:    &corev1.Context{Cluster: GlobalPackagingCluster, Namespace: DefaultNamespace},
+			Identifier: name,
+			Plugin:     plugin,
+		},
+		Name:            name,
+		Description:     DefaultDescription,
+		NamespaceScoped: false,
+		Type:            "helm",
+		Url:             DefaultRepoURL,
+		Interval:        DefaultRepoInterval,
+		TlsConfig:       nil,
+		Auth:            nil,
+		CustomDetail:    nil,
+		Status:          defaultRepoStatus,
+	}
+}
```

Comment:
Thanks for doing the tests here following the existing pattern (using `Make*` factory methods etc.)

Reply:
my pleasure :-)